### ISSUE TITLE
RadioButton doesn't zoom well - because it's composed of a solid circle overlayed onto an empty circle

### ISSUE THREAD

**Issue by acherney** [primefaces/primeng#483]
To test, route here:
http://www.primefaces.org/primeng/#/radiobutton

If you CTRL+ or CTRL- your browser while viewing the radio button, you see the dot move around inside the circle like crazy eyes.  
![image](https://cloud.githubusercontent.com/assets/7864691/16018890/6ebe2052-3174-11e6-8aab-41882b074c40.png).  
Zooming doesn't play well with margin and padding of finely tuned overlayed images - especially when small.

A better design would be to not overlay two graphics, but to instead have a separate graphic for the 'selected' radio button (which includes the external circle) and unselected.  This would scale better at different browser zoom levels.    An example of this would be alternating these two graphics from Font Awesome
http://fontawesome.io/icon/circle-o/
http://fontawesome.io/icon/dot-circle-o/

**Comment by davidda** [comment]
I get this even without zooming. Tested in Edge and Firefox:

![grafik](https://cloud.githubusercontent.com/assets/4145824/21580675/68ddacae-d00a-11e6-8233-26855484fcc2.png)

**Comment by acherney** [comment]
I got around it by rewriting the CSS of the radio button.  I use a high border radius to make it a circle, use the background:black/white to be the dot or unselected, use a thick white border to be the white area around the dot (and control the dot size), and use a dark, dense box shadow or an "outline" as the outline.  This gets rid of the overlay aspect and the reported problem.  Looks pretty slick, too.
On Jan 1, 2017, at 4:10 AM, David Dahlen <notifications@github.com<mailto:notifications@github.com>> wrote:


I get this even without zooming. Tested in Edge and Firefox:

[grafik]<https://cloud.githubusercontent.com/assets/4145824/21580675/68ddacae-d00a-11e6-8233-26855484fcc2.png>

-
